src/Calculators/Dominion/MilitaryCalculator.php

```diff
@@ -376,7 +376,8 @@ protected function getUnitPowerFromLandBasedPerk(Dominion $dominion, Unit $unit,
         $landType = $landPerkData[0];
         $ratio = (int)$landPerkData[1];
         $max = (int)$landPerkData[2];
-        $constructedOnly = $landPerkData[3];
+        $constructedOnly = false;
+        //$constructedOnly = $landPerkData[3]; todo: implement for Nox?
```

Comment:
Can probably be written as `$constructedOnly = $landPerkData[3] ?? false;`?

Reply:
Maybe with @$landPerkData[3]

Reply:
Nooo, bad Fett! No error suppression tokens!! 😰